# Generated by Django 4.2.3 on 2023-07-17 19:46

import django.contrib.auth.validators
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('inventario', '0009_remove_producto_bodegas_alter_perfilusuario_username'),
    ]

    operations = [
        migrations.AddField(
            model_name='producto',
            name='bodegas',
            field=models.ManyToManyField(related_name='productos', to='inventario.bodega'),
        ),
        migrations.AlterField(
            model_name='perfilusuario',
            name='username',
            field=models.CharField(error_messages={'unique': 'A user with that username already exists.'}, help_text='', max_length=150, unique=True, validators=[django.contrib.auth.validators.UnicodeUsernameValidator()], verbose_name='Usuario'),
        ),
    ]
